Art Gallery/Art Gallery/PaintingModel.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  PaintingModel.swift
+//  Art Gallery
+//
+//  Created by Moses Robinson on 1/14/19.
+//  Copyright © 2019 Moses Robinson. All rights reserved.
+//
+
+import UIKit
+
+class PaintingModel: NSObject, UITableViewDataSource, PaintingTableViewCellDelegate {
+    
+    override init() {
+        super.init()
+        self.imageLoader()
+    }
+    
+    func imageLoader() {
+        for i in 1...12 {
+            let imageName = "Image\(i)"
+            guard let image = UIImage(named: imageName) else {return}
+            let painting = Painting(image: image)
+            
+            paintings.append(painting)
+        }
+    }
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return paintings.count
```

Comment:
You need to assign the `tableView` to your `tableView` variable. Which you were declared it on line 62. So you can add this line in one of tableview's `dataSource` functions
```self.tableView = tableView```